crowbar_framework/app/models/cinder_service.rb

```diff
@@ -205,6 +205,20 @@ def apply_role_pre_chef_call(old_role, role, all_nodes)
     # the VIP of the cluster to be setup
     allocate_virtual_ips_for_any_cluster_in_networks(controller_elements, vip_networks)
 
+    # Generate secrets uuid for libvirt rbd backend
+    proposal = ProposalObject.find_data_bag_item "crowbar/bc-cinder-default"
```

Comment:
Yeap, having this attribute in proposal with other cinder backend configuration, it would be the best option.
1. We have to generate UUID for rbd backends, every time when new backend is added.
2. UUID shouldn't be changed after backend creation.
3. We would like to have UUIDs for rbd backends before `cinder.conf` will be generated.
4. Some persistent place for saving this option would be nice.
5. Normally we would ask for this attributes in UI but this will complicate process of adding new rbd backens.